packages/payment-processor/src/contracts/ERC20Contract.ts

```diff
@@ -0,0 +1,297 @@
+/* tslint:disable */
+/* eslint-disable spellcheck/spell-checker */
+/* Generated by ts-generator ver. 0.0.8 */
+
+import { Contract, ContractTransaction, EventFilter, Signer } from 'ethers';
+import { Listener, Provider } from 'ethers/providers';
+import { BigNumber, BigNumberish, Interface } from 'ethers/utils';
+import { TransactionOverrides, TypedEventDescription, TypedFunctionDescription } from '.';
+
+interface ERC20ContractInterface extends Interface {
```

Comment:
In the previous types, we tend to use the prefix `I-` instead of the suffix `-Interface`.

But I am open to keep it like this.